sys/kern/kern_priv.c

```diff
@@ -94,6 +94,11 @@ SYSCTL_INT(_security_bsd, OID_AUTO, unprivileged_read_msgbuf,
     CTLFLAG_RW, &unprivileged_read_msgbuf, 0,
     "Unprivileged processes may read the kernel message buffer");
 
+static int	unprivileged_inode_gen = 0;
```

Comment:
Zero initialization is not needed.  All global variables are by definition zeroed.